sdk/core/perf/src/program.cpp

```diff
@@ -37,14 +35,17 @@ inline Azure::Perf::TestMetadata const* GetTestMetadata(
   argagg::parser argParser;
   auto args = argParser.parse(argc, argv, true);
 
-  auto testName = std::string(args.pos[0]);
-
-  for (auto& test : tests)
+  if (!args.pos.empty())
```

Comment:
Do we support empty args?  Should that default to printing "/?"?

Reply:
Possibly. What happens here is that if you specify the test without providing any arguments, the test app would abort() without printing a diagnostic. This allows the app to print a meaningful diagnostic if you screw up the command line.

Reply:
This is all about discoverability. If you run azure-perf-test with no test name, it used to pop up a "abort() has been called" dialog because the `args.pos[0]` call threw a range exception.

By returning null from GetTestMetaData, it puts the code in the same codepath as is done for an unknown test name, which lists the tests.